apps/dashboard/mixins.py

```diff
@@ -56,3 +57,24 @@ def post(self, request, *args, **kwargs):
 
         return redirect('dashboard-project-list',
                         organisation_slug=self.organisation.slug)
+
+
+class DashboardModRemovalMixin:
+    def post(self, request, *args, **kwargs):
+        if 'submit_action' in request.POST:
+            pk = int(request.POST['moderator_pk'])
+            user = get_object_or_404(User, pk=pk)
+            project = self.get_object()
+            can_edit = request.user.has_perm('a4projects.edit_project',
+                                             project)
```

Comment:
Better check for `meinberlin_organisations.initiate_project`, as moderator should not be able to add moderators.